_data/people.yml

```diff
@@ -5,9 +5,10 @@ corinnekrych:
   update-description: <a href="https://github.com/corinnekrych">corinnekrych</a> <span>opened</span> pull request <a href="https://github.com/aerogear/aerogear-ios-oauth2/pull/17">aerogear/aerogear-ios-oauth2#17</a>
   avatar: https://avatars1.githubusercontent.com/u/1395710
   url: corinnekrych.org
-  location-city: City
+  twitter: corinnekrych
+  location-city:
   location-country: France
-  description: Senior technical consultant with 15 years experience, 12 companies, worked in different environments and methodologies. Enthusiastic, curious and skilled. Pragmatic thinker I will understand requirements and turn them into quality software and useful documentation.
+  description: Senior technical consultant with 15 years experience, 12 companies, worked in different environments and methodologies. Enthusiastic, curious and skilled. Pragmatic thinker I will understand requirements and turn them into quality software and useful documentation. Corinne leads iOS team in AeroGear.
```

Comment:
to be replaced with more friendly content:
"Code is craft and collaboration is key to success. I love chatting the latest tech trends at coffee break: female geek. Mobile is my playground, AeroGear my open source family and iOS my platform by heart."